server/request.js

```diff
@@ -7,6 +7,7 @@ let cookieParser = require("cookie-parser");
 let cookieSession = require("cookie-session");
 
 let version = require("../package").version;
+const gitRevSync = require("git-rev-sync").long();
```

Comment:
@ericsuhn can you rename this to `commit`...then I think this should be good to land